istio/oam.go

```diff
@@ -20,8 +20,6 @@ func (istio *Istio) HandleComponents(comps []v1alpha1.Component, isDel bool, kub
 
 	compFuncMap := map[string]CompHandler{
 		"IstioMesh":            handleComponentIstioMesh,
-		"VirtualService":       handleComponentVirtualService,
-		"EnvoyFilterIstio":     handleComponentEnvoyFilter,
```

Comment:
Maybe a dumb question, but here it means we don't handle Istio VirtualService and EnvoyFilter anymore?

Reply:
We have "Core" components as well as "Istio" components in adapter. "IstioMesh" is a core component. Core component is something for which only we have logic and Istio does not understand that. Up untill now, we were also supporting "VirtualService and DestinationRule" as core but there is not need since it is kind of duplication as Istio supports those. So that is why these are being removed.